src/main/scala/org/hammerlab/guacamole/reads/MappedRead.scala

```diff
@@ -54,21 +54,27 @@ case class MappedRead(
   override val isMapped = true
   override def asMappedRead = Some(this)
 
+
   /**
    * Number of mismatching bases in this read. Does *not* include indels: only looks at read bases that align to a
    * single base in the reference and do not match it.
    *
    * @param referenceContigSequence the reference sequence for this read's contig
    * @return count of mismatching bases
    */
+
+  var countOfMismatches = -1
```

Comment:
given the mixup above, I'd like to change this pattern somehow to ensure that this isn't referenced before it's initialized